lib/txgh/handlers/triggers/handler.rb

```diff
@@ -0,0 +1,67 @@
+module Txgh
+  module Handlers
+    module Triggers
+      class Handler
+
+        class << self
+          def handle_request(request, logger)
+            handle_safely do
+              config = Txgh::Config::KeyManager.config_from_project(
+                request.params.fetch('project_slug')
+              )
+
+              handler_for(config, request, logger).execute
+            end
+          end
+
+          private
+
+          include ResponseHelpers
+
+          def handle_safely
+            yield
+          rescue => e
+            respond_with_error(500, "Internal server error: #{e.message}")
+          end
+
+          def handler_for(config, request, logger)
+            new(
+              project: config.transifex_project,
+              repo: config.github_repo,
+              branch: request.params.fetch('branch'),
+              resource_slug: request.params.fetch('resource_slug'),
+              logger: logger
+            )
+          end
+        end
+
+        include ResponseHelpers
```

Comment:
could define this on `ResponseHelpers` to only include it once and make it available as class or instance methods:

```ruby
def self.included(base)
  base.extend(self)
end
```